api/prisma/schema.prisma

```diff
@@ -15,139 +15,124 @@ type File {
+  isApisMicroservicesCert      Boolean                       @default(false)
+  isBackEndCert                Boolean                       @default(false)
+  isBanned                     Boolean                       @default(false)
+  isCheater                    Boolean                       @default(false)
+  isClassroomAccount           Boolean                       @default(false)
+  isCollegeAlgebraPyCertV8     Boolean                       @default(false)
+  isDataAnalysisPyCertV7       Boolean                       @default(false)
+  isDataVisCert                Boolean                       @default(false)
+  isDonating                   Boolean                       @default(false)
+  isFoundationalCSharpCertV8   Boolean                       @default(false)
+  isFrontEndCert               Boolean                       @default(false)
+  isFrontEndLibsCert           Boolean                       @default(false)
+  isFullStackCert              Boolean                       @default(false)
+  isHonest                     Boolean                       @default(false)
+  isInfosecCertV7              Boolean                       @default(false)
+  isInfosecQaCert              Boolean                       @default(false)
+  isJsAlgoDataStructCert       Boolean                       @default(false)
+  isJsAlgoDataStructCertV8     Boolean                       @default(false)
+  isMachineLearningPyCertV7    Boolean                       @default(false)
+  isQaCertV7                   Boolean                       @default(false)
+  isRelationalDatabaseCertV8   Boolean                       @default(false)
+  isRespWebDesignCert          Boolean                       @default(false)
+  isSciCompPyCertV7            Boolean                       @default(false)
+  keyboardShortcuts            Boolean                       @default(false)
+  lastUpdatedAtInMS            Int
+  linkedin                     String?
+  location                     String?
+  name                         String?
+  needsModeration              Boolean                       @default(false)
+  newEmail                     String?
+  partiallyCompletedChallenges PartiallyCompletedChallenge[]
+  picture                      String?
   portfolio                    Portfolio[]
-  profileUI                    ProfileUI? // Undefined
-  progressTimestamps           Json? // ProgressTimestamp[] | Null[] | Int64[] | Double[] - TODO: NORMALIZE
+  profileUI                    ProfileUI
+  progressTimestamps           Int[]
   /// A random number between 0 and 1.
   ///
   /// Valuable for selectively performing random logic.
-  rand                         Float?
-  savedChallenges              SavedChallenge[] // Undefined | SavedChallenge[]
+  rand                         Float
+  savedChallenges              SavedChallenge[]
   sendQuincyEmail              Boolean
-  theme                        String? // Undefined
```

Comment:
This is no longer needed since the theme is being handled on the client using local storage.